handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -98,12 +100,26 @@
         }
         DESTROYED_UPDATER = destroyedUpdater;
 
+        Method getUseCipherSuitesOrderMethod = null;
+        Method setUseCipherSuitesOrderMethod = null;
         Class<?> sniHostNameClass = null;
         Method getAsciiNameMethod = null;
         Method getServerNamesMethod = null;
         Method setServerNamesMethod = null;
         if (PlatformDependent.javaVersion() >= 8) {
             try {
+                getUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("getUseCipherSuitesOrder");
+                SSLParameters parameters = new SSLParameters();
+                @SuppressWarnings("unused")
+                Boolean order = (Boolean) getUseCipherSuitesOrderMethod.invoke(parameters);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "order" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1481)